packages/lwc-language-server/src/metadata-utils/static-resources-util.ts

```diff
@@ -33,7 +33,10 @@ export async function updateStaticResourceIndex(updatedFiles: FileEvent[], { wor
         }
     }
     if (didChange) {
-        return processStaticResources(workspaceRoots[0], writeConfigs);
+        for (const ws of workspaceRoots) {
+            processStaticResources(ws, writeConfigs);
+        }
+        return;
```

Comment:
Does processStaticResources return anything?

Reply:
This used to return the result from processStaticResources(), so just making sure callers of this function aren't expecting something in the return value

Reply:
processStaticResources does a fs.writeFIle so no real return value. empty return in updateStaticResourceIndex should be ok.